test/unit/smart_proxy_test.rb

```diff
@@ -59,11 +59,15 @@ class SmartProxyTest < ActiveSupport::TestCase
   end
 
   context "#version" do
+    def setup
+      Rails.cache.clear
+    end
+
     test "should succeed" do
       proxy = smart_proxies(:one)
-      fake_version = {:version => '1.11'}
+      fake_version = {'version' => '1.11'}
```

Comment:
You can make it expect the cache to contain the version if you don't want to clear the cache for this test.